theories/Init/.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+((coq-mode . ((eval . (let ((default-directory (locate-dominating-file
+						buffer-file-name ".dir-locals.el")))
+			(make-local-variable 'coq-prog-args)
+			(setq coq-prog-args `("-emacs" "-R" ,(expand-file-name "..") "Coq" "-noinit")))))))
```

Comment:
Because `-noinit` is required for files in the `Init` directory, and wrong for every other file.